etc/ci/add_upstream.sh

```diff
@@ -1,11 +1,10 @@
 #!/bin/bash
 
 # in case we're run from out of git repo
-DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
-pushd "$DIR" 1>/dev/null
+DIR="$(dirname "$(realpath "${BASH_SOURCE[0]}" )")"
+pushd "$DIR" 1>/dev/null || exit 1
+trap popd EXIT
 
 echo "Adding remote..."
-git remote add upstream git://github.com/HoTT/book.git
+git remote add upstream https://github.com/HoTT/book
```

Comment:
Are you sure this doesn't break the authentication method?  What's the reason for this change?

Reply:
The reason is that https://github.com/actions/checkout, by default, would not set up a SSH key.